conf/messages.en

```diff
@@ -99,7 +99,7 @@ amlSupervisor.AssociationOfInternationalAccountants = Association of Internation
 amlSupervisor.AssociationOfCharteredCertifiedAccountants = Association of Chartered Certified Accountants
 amlSupervisor.CharteredInstituteOfLegalExecutives = Chartered Institute of Legal Executives
 amlSupervisor.CharteredInstituteOfManagementAccountants = Chartered Institute of Management Accountants
-amlSupervisor.CharteredInstituteOfTaxation = Chartered Institute of Taxation (CIOT)
+amlSupervisor.CharteredInstituteOfTaxation = Chartered Institute of Taxation
```

Comment:
The ticket has more to it Patrick.

The ordering needs to match what's in the ticket, and a missing one needs adding as well.